app/boot_control.py

```diff
@@ -0,0 +1,50 @@
+from abc import ABC
+from pathlib import Path
+
+
+class BootControlMixinInterface(ABC):
+    """
+    platform neutral boot control interface
+    """
+
+    _boot_control = None
+    _ota_status = None
+    _mount_point = None
+
+    def initialize_ota_status(self):
+        pass
+
+    def write_standby_ota_status(self, status):
```

Comment:
`store_standby_ota_status`, I prefer.
I'd like to use corresponding terms:
write <-> read
load <-> store (retrieve <-> store)
get <-> set
get <-> put (direction)
enter <-> leave
start <-> stop
begin <-> end
etc.